src/IncrementalChecker.ts

```diff
@@ -134,20 +137,22 @@ class IncrementalChecker {
    * Check for vue-loader in user's project node_modules.
    */
   hasVueLoader(): boolean {
-    const projectPath = path.dirname(this.programConfigFile);
+    if (this.vueLoaderResolved !== undefined) return this.vueLoaderResolved;
     
     try {
-      Resolve.sync('vue-loader', { basedir: projectPath });
-      return true;
+      Resolve.sync('vue-loader', { basedir: this.basedir });
+      this.vueLoaderResolved = true;
     } catch (e) {
       // vue-loader doesn't exist, so return false.
-      return false;
+      this.vueLoaderResolved = false;
     }
+
+    return this.vueLoaderResolved;
   }
 
   nextIteration() {
     if (!this.watcher) {
-      this.watcher = new FilesWatcher(this.watchPaths, ['.ts', '.tsx']);
+      this.watcher = new FilesWatcher(this.watchPaths, ['.ts', '.tsx', '.vue']);
```

Comment:
Could the `.vue` be conditional based on whether vue is in the mix or not please?